config/pepperConfig.js.ctmpl

```diff
@@ -14,6 +14,17 @@ var DDP_ENV = {
     tcellbaseurl: "https://us.agent.tcell.insight.rapid7.com/api/v1"
 };
 
+#Log levels: 1 - debug; 2 - info; 3 - warning; 4 - error. E.g. 1 includes debug level and higher, 2 includes info and higher, etc.
+{{if eq $environment "dev"}}
+    DDP_ENV['logLevel'] = 1;
+{{else if eq $environment "test"}}
+    DDP_ENV['logLevel'] = 1;
+{{else if eq $environment "staging"}}
+    DDP_ENV['logLevel'] = 1;
+{{else if eq $environment "prod"}}
+    DDP_ENV['logLevel'] = 2;
+{{end}}
```

Comment:
Decided to setup LogLevel for all apps in one place. If you have another opinion about LogLevel for some env, please, let me know.
cc @zyme